src/app.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const artistRouter = require('./routes/artist');
+const albumRouter = require('./routes/album');
+const songRouter = require('./routes/song');
+
+const app = express();
+
+app.use(express.json());
+
+app.use('/artist', artistRouter);
+
+app.use('/album', albumRouter);
+
+app.use('/song', songRouter);
```

Comment:
These could all be bunched together as given its such a small file it wont negatively affect readability